enos/modules/run_workloads/main.tf

```diff
@@ -23,8 +28,27 @@ resource "enos_local_exec" "wait_for_nomad_api" {
   scripts = [abspath("${path.module}/scripts/wait_for_nomad_api.sh")]
 }
 
+resource "enos_local_exec" "get_nodes" {
+  environment = local.nomad_env
+
+  inline = ["nomad node status -json | jq '[.[] | select(.Status == \"ready\")] | length'"]
+}
+
+resource "enos_local_exec" "get_jobs" {
+  environment = local.nomad_env
+
+  inline = ["nomad job status| awk '$4 == \"running\" {count++} END {print count+0}'"]
+}
+
+resource "enos_local_exec" "get_allocs" {
+  environment = local.nomad_env
+
+  inline = ["nomad alloc status -json | jq '[.[] | select(.ClientStatus == \"running\")] | length'"]
+}
```

Comment:
I think I'm a little confused as to the purpose of these resources. If `enos_local_exec.workloads` depends on these, aren't they always going to be 0? Or are we assuming there could be other workloads running on the cluster already?

Reply:
Im thinking about the fact that enos does not accept functions in variable inputs for te steps, so if we were to reuse the run_workloads modules and then the test_cluster_health, we need the output of run_workloads to account for previously running allocs, because we cant add them to pass to test_cluster_health. Maybe Im overthinking it?